### PR TITLE
Add NeuVector Monitor chart version 2.7.3

### PR DESCRIPTION
**Pull request by selvamt94**
## Problem
NeuVector Monitor chart 102.0.8+up2.7.3 need to be added to Rancher feature charts version 2.7 (release-v2.7). We would like to include this chart in Rancher Manager release v2.7.11 otherwise this need to be released as OOB.
## Solution
new chart has been added

### Manual Testing
Tested fresh install of the NeuVector Monitor chart 102.0.8+up2.7.3 and upgraded from previous version on RKE and RKE2 cluster.
## QA Testing Considerations
Tested fresh install of the NeuVector Monitor chart 102.0.8+up2.7.3 and  upgraded from previous version on RKE and RKE2 cluster.

### CODE REVIEW

**Comment by adamkpickering** [comment]
Looks good to me, though I'm going to hold off on approval for the reasons stated in https://github.com/rancher/charts/pull/3460#issuecomment-1942782423.

**Comment by adamkpickering** [comment]
As @nicholasSUSE confirmed in https://github.com/rancher/charts/pull/3460#pullrequestreview-1883842481, please target `dev-v2.7.11` rather than `dev-v2.7`.

File: packages/neuvector-monitor/package.yaml
Comment (lucasmlp) [inline packages/neuvector-monitor/package.yaml:1]: @selvamt94 just out of curiosity. You bumped two minor versions here so the package version would match the neuvector one?

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Comment by lucasmlp** [comment]
@nicholasSUSE and @adamkpickering 
@selvamt94 opened a different [PR](https://github.com/rancher/charts/pull/3475) to the `dev-v2.7.11`

**Pull request merged**